public/scripts/loginHandler.js

```diff
@@ -21,7 +21,10 @@ function updateLobby(usernames) {
   lobbyList.innerHTML = ''
   usernames.forEach(username => {
     const listItem = document.createElement('li')
-    const text = (username === myUsername) ? `> ${username}` : username
+    let text = username
+    if (username === myUsername) {
+      listItem.style.color = 'gold'
```

Comment:
it'd be better to do something like
```js
listItem.classList.add('myName') // or listItem.className = 'myName'
```
then have styling for that class in the .css file. That'd let us adjust many bits of style if we want to do other things to it